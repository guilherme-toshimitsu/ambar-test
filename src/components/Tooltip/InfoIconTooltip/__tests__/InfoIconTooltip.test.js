import React from 'react';
import { render } from '@testing-library/react';
import IconInfoTooltip from '../';

describe('<IconInfoTooltip> Composition of Tooltip', () => {
  it('renders', () => {
    const container = render(<IconInfoTooltip title={'teste'} />);
    expect(container.firstChild).toMatchSnapshot();
  });
});
