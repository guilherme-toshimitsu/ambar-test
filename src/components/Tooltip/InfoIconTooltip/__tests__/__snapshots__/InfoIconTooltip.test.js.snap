// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<IconInfoTooltip> Composition of Tooltip renders 1`] = `undefined`;
